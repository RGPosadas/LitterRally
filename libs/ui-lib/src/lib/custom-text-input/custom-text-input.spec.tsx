import React from 'react';
import { render } from '@testing-library/react';

import CustomTextInput from './custom-text-input';

describe('CustomTextInput', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<CustomTextInput />);
    expect(baseElement).toBeTruthy();
  });
});
