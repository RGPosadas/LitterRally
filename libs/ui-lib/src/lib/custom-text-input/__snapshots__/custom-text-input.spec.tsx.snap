// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomTextInput should match snapshot 1`] = `
<div
  className="custom-text-input-container error"
>
  <label
    className="custom-text-input-label"
    htmlFor="text"
  >
    Text
  </label>
  <input
    className="custom-text-input input-border error"
    data-testid="custom-text-input"
    id="text"
    name="text"
    onChange={
      [MockFunction] {
        "calls": Array [
          Array [
            SyntheticEvent {
              "_dispatchInstances": null,
              "_dispatchListeners": null,
              "_targetInst": null,
              "bubbles": null,
              "cancelable": null,
              "currentTarget": [Function],
              "defaultPrevented": null,
              "dispatchConfig": null,
              "eventPhase": null,
              "isDefaultPrevented": [Function],
              "isPropagationStopped": [Function],
              "isTrusted": null,
              "nativeEvent": null,
              "target": null,
              "timeStamp": [Function],
              "type": null,
            },
          ],
        ],
        "results": Array [
          Object {
            "type": "return",
            "value": undefined,
          },
        ],
      }
    }
    type="text"
    value=""
  />
  <span
    className="error-message"
  >
    Text is not valid.
  </span>
</div>
`;
